x-pack/solutions/security/plugins/security_solution/public/asset_inventory/components/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { FC, PropsWithChildren } from 'react';
+import { GetStarted } from './get_started';
+import { useAssetInventoryContext } from '../../context';
+import { Loading } from '../loading';
+
+/**
+ * This component serves as a wrapper to render appropriate onboarding screens
+ * based on the current onboarding status. If no specific onboarding status
+ * matches, it will render the child components.
+ */
+export const Onboarding: FC<PropsWithChildren> = ({ children }) => {
```

Comment:
`Onboarding` is pretty generic, can sound like a reuseable onboarding component, can we rename to `AssetInventoryOnboarding`

Reply:
I think it makes sense to rename it to `AssetInventoryOnboarding`, it's indeed a component very coupled to asset inventory and renaming also makes it easier to find on the IDE.